lib/action_subscriber.rb

```diff
@@ -80,6 +80,10 @@ def self.start_subscribers
     print_subscriptions
   end
 
+  def self.stop_subscribers!
+    @route_set.cancel_consumers! if @route_set
```

Comment:
I wouldn't be accessing the class-instance variable. If you make a method for it, you can incorporate the guard clause in it.